mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -4671,8 +7169,22 @@
 													"});",
 													"",
 													"pm.test(\"Validate schema\", function () {",
-													"    tv4.addSchema(\"JSONAPI.json\", JSON.parse(pm.variables.get(\"schema_jsonapi_content\")));",
-													"    pm.expect(tv4.validate(response, JSON.parse(pm.environment.get(\"schema_jsonapi_content\")))).to.equal(true, \"Schema validation error: \" + JSON.stringify(tv4.error));",
+													"    tv4.addSchema(\"schema_packageCollectionItem.json\", JSON.parse(pm.variables.get(\"schema_packageCollectionItem\")));",
```

Comment:
Is it posisble to add all necessary schemas once during set-up?

Reply:
Unfortunately no, I have not found information in the documentation but found this feature experimentally.